### ISSUE TITLE
Always send JSON

### ISSUE THREAD

**Issue by oliverjam** [fac19/week7-BGJK#30]
https://github.com/fac19/week7-BGJK/blob/621c334566a09c73530d761c109161583bc4e242/middleware/error.js#L8

As someone building an app using this API it would probably break my code to suddenly get a string response when an error occurred. It would be better if your responses were always JSON, so I can handle them consistently in my code.

E.g. my frontend code might look like this:

```js
fetch("https://bgjk.com/questions")
  .then(res => res.json())
  .then(json => updateUISomehow(json)
  .catch(console.error)
```

I need to know what type the response is: the `res.json()` there will throw an error if the server returns a string.

**Comment by oliverjam** [comment]
Is this fixed? I can't see any recent commits. Probably a good idea to keep issues open until the commit/PR that references it is merged

**Comment by CampbellDocherty** [comment]
Sorry, we committed forgot to push

**Comment by oliverjam** [comment]
@CampbellDocherty it's usually easiest to let the commit close the issue for you (add `fixes #30` to the message and when you push it'll auto-close the issue)
